modelopt/torch/nas/plugins/megatron_hooks/__init__.py

```diff
@@ -16,4 +16,3 @@
 
 from .base_hooks import *
 from .base_hooks_analysis import *
-from .megatron_hooks import *
```

Comment:
We dont want to remove it but rather guard the import so if megatron is not present, this does not raise an error. Like so:

```python
from modelopt.torch.utils import import_plugin

with import_plugin("megatron_hooks"):
    from .megatron_hooks import *
```